pkg/manifest/loader/account_test.go

```diff
@@ -63,36 +70,125 @@ func TestValidAccounts(t *testing.T) {
 		},
 	}
 
-	v, err := parseAccounts(&Context{}, []persistence.Account{acc, acc2})
-	assert.NoError(t, err)
+	//account 3 has UUID defined as env var
+	envUUID := uuid.New().String()
+	t.Setenv("ACC_3_UUID_ENV_VAR", envUUID)
+	acc3 := persistence.Account{
+		Name: "name3",
+		AccountUUID: persistence.AccountUUID{
+			Type:  persistence.TypeEnvironment,
+			Value: "ACC_3_UUID_ENV_VAR",
+		},
+		OAuth: persistence.OAuth{
+			ClientID: persistence.AuthSecret{
+				Name: "SECRET",
+			},
+			ClientSecret: persistence.AuthSecret{
+				Name: "SECRET",
+			},
+			TokenEndpoint: nil,
+		},
+	}
 
-	assert.Equal(t, v, map[string]manifest.Account{
-		"name": {
-			Name:        "name",
-			AccountUUID: uuid.MustParse(acc.AccountUUID),
-			ApiUrl: &manifest.URLDefinition{
-				Type:  manifest.ValueURLType,
-				Value: "https://example.com",
+	t.Run("full account", func(t *testing.T) {
```

Comment:
note: I found it nicer to have these organized as individual tests. I still keep the "load all" as a dedicated one.